### PR TITLE
provide more details for building steps

### PR DESCRIPTION
**Pull request by adolphlwq**
1. add more detailed steps for  building code
2. fix typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,28 +95,32 @@ Download from [Bintray](https://dl.bintray.com/kompose/kompose/latest/kompose-wi
...
+3. Set `GOPATH` correctly or click [SettingGOPATH](https://github.com/golang/go/wiki/SettingGOPATH) for details
 
-- You need `make`
-- You need `go` v1.6 or later.
-- If your working copy is not in your `GOPATH`, you need to set it accordingly.
-
-You can either build via the Makefile:
-
+__Steps:__
+- clone code
```
Comment (cdrage) [inline README.md]: Change to: `1. Clone repository`

File: README.md [hunk 2]
```diff
@@ -95,28 +95,32 @@ Download from [Bintray](https://dl.bintray.com/kompose/kompose/latest/kompose-wi
...
+```console
+$ git clone https://github.com/kubernetes/kompose.git $GOPATH/src/github.com/kubernetes/kompose
+```
+- build by make
```
Comment (cdrage) [inline README.md]: Change to: 2. Build with `make`

File: README.md [hunk 3]
```diff
@@ -95,28 +95,32 @@ Download from [Bintray](https://dl.bintray.com/kompose/kompose/latest/kompose-wi
...
 ```console
 $ make bin
 ```
-
-Or `go build`:
-
+- build by go
```
Comment (cdrage) [inline README.md]: Change to: Or build with `go`

File: README.md [hunk 4]
```diff
@@ -95,28 +95,32 @@ Download from [Bintray](https://dl.bintray.com/kompose/kompose/latest/kompose-wi
...
 $ go build -o kompose main.go
 ```
-
-To create a multi-platform binary, use the `cross` command via `make`:
-
+- create a multi-platform binary
 ```console
 $ make cross
 ```
+- do not forget test
```
Comment (cdrage) [inline README.md]: Change to `Test your changes`

File: README.md [hunk 5]
```diff
@@ -95,28 +95,32 @@ Download from [Bintray](https://dl.bintray.com/kompose/kompose/latest/kompose-wi
...
-Or `go build`:
-
+- build by go
 ```console
 $ go build -o kompose main.go
 ```
-
-To create a multi-platform binary, use the `cross` command via `make`:
-
+- create a multi-platform binary
```
Comment (cdrage) [inline README.md]: This should be removed as we already explain how to create a multi-platform binary :)

File: README.md (hunk 4 above)
Comment (adolphlwq) [reply README.md]: OK

**Comment by adolphlwq** [comment]
@cdrage Has update

**Comment by cdrage** [comment]
Thank you for your contribution, this LGTM! 👍

**Pull request merged**